docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -86,67 +86,73 @@ In the event of a total active region loss, the following data will be lost:
-- The customers operating their Camunda 8 setup are responsible for detecting a regional failure and executing the [operational procedure](./../../operational-guides/multi-region/dual-region-ops.md).
-
-## Limitations
-
-- We recommend using a Kubernetes dual-region setup, with [Camunda Helm chart](/self-managed/setup/install.md) installed in two Kubernetes clusters.
-  - Using alternative installation methods (for example, with docker-compose) is not covered in our documentation.
-- Looking at the whole Camunda platform, it's **active-passive**, while some key components are active-active.
-  - There's always one active and one passive region for serving active user traffic.
-  - Serving traffic to both regions will result in a detachment of the components and users potentially observing different data in Operate and Tasklist.
-- Identity is not supported.
-  - Multi-tenancy does not work.
-  - Role Based Access Control (RBAC) does not work.
-- Optimize is not supported.
-  - This is due to Optimize depending on Identity to work.
-- Connectors can be deployed alongside but ensure to understand idempotency based on [the described documentation](../../../components/connectors/use-connectors/inbound.md#creating-the-connector-event).
-  - in a dual-region setup, you'll have two connector deployments and using message idempotency is of importance to not duplicate events.
-- Zeebe cluster scaling is not supported.
-- Web-Modeler is a standalone component and is not covered in this guide.
-  - Modeling applications can operate independently outside of the automation clusters.
-
-## Considerations
-
-Multi-region setups in itself bring their own complexity. The following items are such complexities and are not considered in our guides.
-You should familiarize yourself with those before deciding to go for a dual-region setup.
-
-- Managing multiple Kubernetes clusters and their deployments across regions
-- Monitoring and alerting
-- Increased costs of multiple clusters and cross-region traffic
-- Data consistency and synchronization challenges (for example, brought in by the increased latency)
-  - Bursts of increased latency can already have an impact
-- Managing DNS and incoming traffic
+## Requirements and Limitations
+
+### Minimum Versions
+
+| **Component**       | **Elasticsearch** | **Operate** | **Tasklist** | **Zeebe** | **Zeebe Gateway** | **Camunda Helm Chart**                                   |
+| ------------------- | ----------------- | ----------- | ------------ | --------- | ----------------- | -------------------------------------------------------- |
+| **Minimum Version** | 8.9+\*            | 8.5+        | 8.5+         | 8.5+      | 8.5+              | [9.3+](https://github.com/camunda/camunda-platform-helm) |
+
+\* OpenSearch (both managed and self-managed) is not supported for a dual-region setup.
```

Comment:
Wasn't a big fan of this in the past already and wondering whether we should just remove it altogether.
We're doing versioned docs, meaning the 8.5 stuff isn't really relevant for 8.6.
We have an extra document on supported environments stating e.g. Elasticsearch version.

Anything 8.5 and 8.6 are compatible. 9.3 is even 8.4 which may even be more confusing to customers.

OpenSearch would still need to be mentioned somewhere.

Reply:
we don't rely on anything DB specific and just rely on the version stated in the supported environments.
With the only exception of OpenSearch and that mostly due to tests missings.